vuepress/docs/next/docs/ecr/ecr-bundle-details.md

```diff
@@ -474,36 +474,37 @@ This descriptor enables a content to be created and optionally published via a b
 
       
 ## Categories
-This descriptor contains a *list* of Categories.
+
+This descriptor contains a list of Categories:
```

Comment:
are these one liners for all the descriptors below needed?

Reply:
sigh. no. text for text's sake...